lesson_06/conditionals/conditionals_app/src/main/java/com/codedifferently/lesson6/Lesson6.java

```diff
@@ -61,7 +97,13 @@ public static String convertGpaToLetterGrade(double gpa) {
    * @return The factorial of n.
    */
   public static int computeFactorial(int n) {
-    return 0;
+    int factorial = 1;
+
+    for (int i = 1; i <= n; i++) {
```

Comment:
What if n is negative? This won't work.

Reply:
More specifically, we should throw an exception if `n < 0`, but we haven't covered that yet. I'll allow this for now.